manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Hide LinkedIn Applicants",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
Incremented the version to reflect this new change.

Reply:
I'm using [semantic versioning ](https://docs.npmjs.com/about-semantic-versioning) and following best practices for the Chrome web store according to [this article](https://developer.chrome.com/docs/webstore/publish/#create-your-items-zip-file).